src/components/nav/desktop/NavAudio.tsx

```diff
@@ -37,17 +43,56 @@ const NavAudio = () => {
   // so below null-coalescing is okay
   const { tier } = useSelectTierInfo(account?.user_id ?? 0)
   const audioBadge = audioTierMapPng[tier]
+  const pendingAutoClaims = useSelector(getPendingAutoClaims)
+  const [bubbleType, setBubbleType] = useState<BubbleType>('empty')
+  const claimPromptDelayMs = useRemoteVar(IntKeys.MANUAL_CLAIM_PROMPT_DELAY_MS)
+
+  const goToAudioPage = useCallback(() => {
+    navigate(AUDIO_PAGE)
+  }, [navigate])
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (
+        Object.values(pendingAutoClaims).some(
+          time => time && new Date().getTime() - time > claimPromptDelayMs
+        )
+      ) {
+        setBubbleType('claim')
+      } else if (nonNullTotalBalance && !positiveTotalBalance) {
+        setBubbleType('earn')
+      } else {
+        setBubbleType('empty')
+      }
+    }, 100)
```

Comment:
100 seems aggressive here, since this is running all the time. what if we did 500?

Also I would think that instead of the interval, you could just do this with a useEffect that listens for changes to `pendingAutoClaims` since every time that is updated it would be a new ref

Reply:
see if you like my update that uses `setTimeout`?